exts/cesium.omniverse/cesium/omniverse/extension.py

```diff
@@ -407,3 +412,16 @@ def show_debug_window(self, _menu, value):
             asyncio.ensure_future(self._dock_window_async(self._debug_window))
         elif self._debug_window is not None:
             self._debug_window.visible = False
+
+    def _setup_credits_viewport_frames(self):
+        self._destroy_credits_viewport_frames()
+        self._credits_viewport_frames = [
+            CesiumCreditsViewportFrame(_cesium_omniverse_interface, i) for i in get_viewport_window_instances()
+        ]
+        if self._credits_viewport_controller is not None:
+            self._credits_viewport_controller.broadcast_credits()
+
+    def _destroy_credits_viewport_frames(self):
+        for credits_viewport_frame in self._credits_viewport_frames:
+            credits_viewport_frame.destroy()
+        self._credits_viewport_frames.clear()
```

Comment:
Potential Nitpick: Do you expect to call these multiple times? If not, I'm not sure it's worth breaking them out into separate methods. I can see either way though.

Reply:
I don't expect to call these multiple times in the short term, but probably in the long term as we develop the extension to handle multiple viewports that can be setup, configured, and destroyed on the fly.

Thinking about it, this method should be in the controller. Perhaps it's best to push that task and this question into the larger plan of modularizing extension.py?

Reply:
> Perhaps it's best to push that task and this question into the larger plan of modularizing extension.py?

Yeah, I'm cool with that.